packages/fxa-payments-server/src/store/types.tsx

```diff
@@ -94,43 +103,14 @@ export type CreateSubscriptionError = {
   info?: string;
   statusCode?: number;
 };
-export type CreateSubscriptionFetchState = FetchState<
-  CreateSubscriptionResult,
-  CreateSubscriptionError
->;
 
 export interface UpdateSubscriptionPlanResult {
   subscriptionId: string;
 }
-export type UpdateSubscriptionPlanFetchState = FetchState<
-  UpdateSubscriptionPlanResult,
-  APIError
->;
-
-export type PlansFetchState = FetchState<Array<Plan>, APIError>;
```

Comment:
Now that the `State` type is just `typeof defaultState`, we can use an `APIState` alias to `State["api"]` as a replacement for all of these individual type declarations.